src/core/main.c

```diff
@@ -1598,11 +1599,20 @@ static void initialize_clock(void) {
                  */
                 (void) clock_reset_timewarp();
 
-        r = clock_apply_epoch();
-        if (r < 0)
-                log_error_errno(r, "Current system time is before build time, but cannot correct: %m");
-        else if (r > 0)
+        ClockChangeDirection change_dir;
+        r = clock_apply_epoch(&change_dir);
+        if (r == 0)
+                log_info("Initialized system clock");
```

Comment:
nit: missing period. Also, maybe make this `log_debug`? Finally, the system clock is changed only if `r == 1`, no? In case `r == 0` it's a no-op?

Reply:
so maybe just remove this message altogether, as it would be confusing (says clock was initialized, but nothing was actually done)

Reply:
done